R/get.variable.names.r

```diff
@@ -48,9 +41,20 @@
 #' 1972 1972           ER30094        ER30100           ER30116
 #' 1973 1973           ER30120        ER30126           ER30137
 #' 
+#' > head(fam)
+#'      year heads_avg_hrly_ern tot_fu_mon_inc_ov414
+#' 1968 1968               V337                  V81
+#' 1969 1969               V871                 V529
+#' 1970 1970              V1567                V1514
+#' 1971 1971              V2279                V2226
+#' 1972 1972              V2906                V2852
+#' 1973 1973              V3275                V3256
+#' 
 
 get.variable.names <- function(vars, years, scrape_name=F, varnames = NULL){
```

Comment:
this is never going to pass the CRAN R CMD CHECK. all dependencies need to be specified for the package to be accepted. please remove the `tryCatch` line and add the dependency to  DESCRIPTION as before. sorry if I wasn't clear before. I was asking if you could achieve this without XML (and your answer was *no*, so that's fine).